config/bidderinfo.go

```diff
@@ -276,8 +276,11 @@ func processBidderAliases(aliasNillableFieldsByBidder map[string]aliasNillableFi
 		if err := validateAliases(aliasBidderInfo, bidderInfos, bidderName); err != nil {
 			return nil, err
 		}
+
 		//required for CoreBidderNames function to also return aliasBiddernames
-		openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName)))
+		if err := openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName)), openrtb_ext.BidderName(aliasBidderInfo.AliasOf)); err != nil {
+			return nil, err
+		}
```

Comment:
few points here:

`bidderName`  is string. so no need to typecast it again as string on line 281

`SetAliasBidderName` takes `bidderName` of `openrtb_ext.BidderName` type  as input and then typecast it to string to call `IsBidderNameReserved`.  

Instead we could update, `SetAliasBidderName` to take `bidderName` of string type as input. With this change,
-  we don't have to type cast bidderName from `string` to `openrtb_ext.BidderName` type while calling  `SetAliasBidderName` on line 281
- also reduces type casting count from 3 to 1
  


```

- if err := openrtb_ext.SetAliasBidderName(openrtb_ext.BidderName(string(bidderName)), openrtb_ext.BidderName(aliasBidderInfo.AliasOf)); err != nil {
+ - if err := openrtb_ext.SetAliasBidderName(bidderName, openrtb_ext.BidderName(aliasBidderInfo.AliasOf)); err != nil {



- func SetAliasBidderName(aliasBidderName BidderName, parentBidderName BidderName) error {
+ - func SetAliasBidderName(aliasBidderName string, parentBidderName BidderName) error {
	- if IsBidderNameReserved(string(aliasBidderName)) {
        + if IsBidderNameReserved(aliasBidderName) {
		return fmt.Errorf("alias %s is a reserved bidder name and cannot be used", aliasBidderName)
	}
        - coreBidderNames = append(coreBidderNames, aliasBidderName)
        - aliasBidderToParentBidder[aliasBidderName] = parentBidderName
        + aliasBidder := openrtb_ext.BidderName(bidderName) // only type casting needed
        + coreBidderNames = append(coreBidderNames, aliasBidder)
	+ aliasBidderToParentBidder[aliasBidder] = parentBidderName
	return nil
}
```

Reply:
Fixed here - https://github.com/prebid/prebid-server/pull/3048/commits/ebb3adc92258095e69433abb4c5df10dcd6b7725